environment/src/test/java/com/sequenceiq/environment/environment/v1/converter/EnvironmentResponseConverterTest.java

```diff
@@ -175,6 +180,7 @@ void testDtoToSimpleResponse() {
         assertEquals(telemetryResponse, actual.getTelemetry());
         assertEquals(compactRegionResponse, actual.getRegions());
         assertEquals(environment.getParameters().getAwsParametersDto().getS3GuardTableName(), actual.getAws().getS3guard().getDynamoDbTableName());
+        assertAzureParameters(environment.getParameters().getAzureParametersDto(), actual.getAzure());
```

Comment:
Please create a pull request only for Azure

Reply:
What do you mean by that?

Reply:
Clarified: separate test method for azure